packages/amazonq/package.json

```diff
@@ -614,6 +607,45 @@
                 "command": "aws.amazonq.showHistoryInHub",
                 "title": "Show Job Status",
                 "icon": "$(history)"
+            },
+            {
+                "command": "aws.amazonq.selectCustomization",
+                "title": "%AWS.codewhisperer.customization.notification.new_customizations.select%",
+                "category": "%AWS.amazonq.title%"
+            },
+            {
+                "command": "aws.amazonq.gettingStarted",
+                "title": "Try inline suggestion examples",
+                "category": "%AWS.amazonq.title%"
+            },
+            {
+                "command": "aws.amazonq.openChatPanelFromCommandPalette",
+                "title": "Open Chat Panel",
+                "category": "%AWS.amazonq.title%"
+            },
+            {
+                "command": "aws.amazonq.resumeAutoSuggestions",
+                "title": "%AWS.amazonq.resumeAutoSuggestions%",
+                "category": "%AWS.amazonq.title%",
+                "enablement": "!aws.codewhisperer.isSuggestionsEnabled"
+            },
+            {
+                "command": "aws.amazonq.pauseAutoSuggestions",
+                "title": "%AWS.amazonq.pauseAutoSuggestions%",
+                "category": "%AWS.amazonq.title%",
+                "enablement": "aws.codewhisperer.isSuggestionsEnabled"
+            },
+            {
+                "command": "aws.amazonq.resumeAutoScans",
+                "title": "%AWS.amazonq.resumeAutoScans%",
+                "category": "%AWS.amazonq.title%",
+                "enablement": "!aws.codewhisperer.isScanEnabled"
+            },
+            {
+                "command": "aws.amazonq.pauseAutoScans",
```

Comment:
can these be one "toggle" command instead of 2 separate commands? that allows users to define a single keybinding